apps/api/auth/dependencies.py

```
@@ -102,3 +105,62 @@ def require_faculty() -> Callable:
+    if current_user.role == UserRole.SUPER_ADMIN:
+        return current_user
+    if not current_user.organizationId:
+        raise PermissionDeniedError(
+            detail="No active organization. Please select a college first."
+        )
+    return current_user
+
+
+async def require_org_admin(
+    request: Request,
+    current_user: AuthenticatedUser = Depends(require_org_member),
```

Ruff (0.15.10)

[warning] Do not perform function call `Depends` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)